src/App.js

```diff
@@ -1,5 +1,14 @@
+const BridgeController = require('./controller/BridgeController');
+
 class App {
-  play() {}
+  #bridgeController;
+  constructor() {
+    this.#bridgeController = new BridgeController();
+  }
+  play() {
+    this.#bridgeController.start();
+  }
```

Comment:
https://github.com/airbnb/javascript#whitespace--after-blocks
블럭 사이에 빈줄이 있으면 더 좋을 것 같습니다.
```suggestion
  #bridgeController;
  
  constructor() {
    this.#bridgeController = new BridgeController();
  }
  
  play() {
    this.#bridgeController.start();
  }
```